packages/block-library/src/term-description/index.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * Server-side rendering of the `core/term-description` block.
+ *
+ * @package WordPress
+ */
+
+/**
+ * Renders the `core/term-description` block on the server.
+ *
+ * @param array    $attributes Block attributes.
+ * @param string   $content    Block default content.
+ * @param WP_Block $block      Block instance.
+ * @return string Returns the filtered post content of the current post.
+ */
+function render_block_core_term_description( $attributes, $content, $block ) {
+
+	$description = false;
+	if ( is_category() ) {
```

Comment:
`category_description` and `tag_description` are just wrappers of `term_description`, so you can use only that one.